terraform/modules/helm/postgresql/postgresql-helm-chart/values.yaml

```diff
@@ -343,7 +343,7 @@ primary:
       02_create_druid_raw_db.sql: |
         CREATE DATABASE druid_raw;
       03_create_druid_raw_user.sql: |
-        CREATE USER druid_raw WITH ENCRYPTED PASSWORD 'druid_raw';
+        CREATE USER druid_raw WITH ENCRYPTED PASSWORD 'druidraw123';
```

Comment:
Can we define the credentials information in the documentation? in the superset section?